src/template.html

```diff
@@ -7,11 +7,158 @@
+            <button>Comment</button>
+            <div>
+              <h4>likes<span>1</span></h4>
+            </div>
+          </div>
+        </li>
+        <li class="pokemonItem">
+          <img src="./assets/3.svg" alt="" />
+          <div class="likeDiv">
+            <h3>Bykatshoo</h3>
+            <img src="./assets/like.svg" alt="" />
+          </div>
+          <div class="commentDiv">
+            <button>Comment</button>
+            <div>
+              <h4>likes<span>1</span></h4>
+            </div>
+          </div>
+        </li>
+        <li class="pokemonItem">
+          <img src="./assets/4.svg" alt="" />
+          <div class="likeDiv">
+            <h3>Bykatshoo</h3>
+            <img src="./assets/like.svg" alt="" />
+          </div>
+          <div class="commentDiv">
+            <button>Comment</button>
+            <div>
+              <h4>likes<span>1</span></h4>
+            </div>
+          </div>
+        </li>
+        <li class="pokemonItem">
+          <img src="./assets/5.svg" alt="" />
+          <div class="likeDiv">
+            <h3>Bykatshoo</h3>
+            <img src="./assets/like.svg" alt="" />
+          </div>
+          <div class="commentDiv">
+            <button>Comment</button>
+            <div>
+              <h4>likes<span>1</span></h4>
+            </div>
+          </div>
+        </li>
+        <li class="pokemonItem">
+          <img src="./assets/6.svg" alt="" />
+          <div class="likeDiv">
+            <h3>Bykatshoo</h3>
+            <img src="./assets/like.svg" alt="" />
+          </div>
+          <div class="commentDiv">
+            <button>Comment</button>
+            <div>
+              <h4>likes<span>1</span></h4>
+            </div>
+          </div>
+        </li>
+        <li class="pokemonItem">
+          <img src="./assets/7.svg" alt="" />
+          <div class="likeDiv">
+            <h3>Bykatshoo</h3>
+            <img src="./assets/like.svg" alt="" />
+          </div>
+          <div class="commentDiv">
+            <button>Comment</button>
+            <div>
+              <h4>likes<span>1</span></h4>
+            </div>
+          </div>
+        </li>
+        <li class="pokemonItem">
+          <img src="./assets/8.svg" alt="" />
+          <div class="likeDiv">
+            <h3>Bykatshoo</h3>
+            <img src="./assets/like.svg" alt="" />
+          </div>
+          <div class="commentDiv">
+            <button>Comment</button>
+            <div>
+              <h4>likes<span>1</span></h4>
+            </div>
+          </div>
+        </li>
+        <li class="pokemonItem">
+          <img src="./assets/9.svg" alt="" />
+          <div class="likeDiv">
+            <h3>Bykatshoo</h3>
+            <img src="./assets/like.svg" alt="" />
+          </div>
+          <div class="commentDiv">
+            <button>Comment</button>
+            <div>
+              <h4>likes<span>1</span></h4>
+            </div>
+          </div>
+        </li> -->
```

Comment:
- [ ] Please remove any unused code from your codebase for a cleaner and more optimized environment. 🙏 see more tips [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.).